src/3d/qgspointcloudlayerchunkloader_p.cpp

```diff
@@ -89,7 +90,7 @@ QgsPointCloudLayerChunkLoader::QgsPointCloudLayerChunkLoader( const QgsPointClou
       QgsDebugMsgLevel( QStringLiteral( "canceled" ), 2 );
       return;
     }
-    mHandler->processNode( pc, pcNode, mContext );
+    mHandler->processNode( pc, pcNode, mContext, mCoordTrans );
```

Comment:
I would suggest to add the coordinate transform as another member of QgsPointCloud3DRenderContext which is meant to contain all the extra bits of information we need to properly render point cloud